pages/self-org-lead.md

```diff
@@ -1,21 +1,25 @@
 ---
 layout: page-fullwidth
 show_meta: false
-title: "Lead instructor checklist"
+title: "Lead instructor/Host checklist"
 header:
    image_fullwidth: "wood_plank.jpg"
 permalink: "/self-org-lead/"
 ---
 
+**Information on hosting responsibilities for self-organized workshops.**
+
 Usually for a self-organized workshop, the lead instructor is also the host. For instances where this is not the case, the lead instructor and host should decide together how to split these responsibilities. The lead instructor is also responsible for preparing to teach the material - see the [Instructor checklist](/instructor-checklist/) for details.
 
+*If your workshop is centrally-organized, the Data Carpentry workshop coordinator supports workshop hosts by taking care of all non-local workshop logistics. See the [centrally-organized host checklist](/host-checklist/) for hosting responsibilities for centrally-organized workshops.*
 
 - [ ] Make sure that you meet all of the [requirements](/self-organized-workshops/#self-organized-workshop-requirements) to run a self-organized workshop
-- [ ] Decide on your [curriculum](http://www.datacarpentry.org/workshops/)  
+- [ ] Decide on your [curriculum](/workshops/)  
 - [ ] [Recruit co-instructor(s)](/email-templates/#recruiting-co-instructors)  
 - [ ] [Request a self-organized workshop](https://amy.software-carpentry.org/workshops/dc/request/)  
 - [ ] Find and book a venue  
-- [ ] Ensure venue is [accessible](/accessibility/)  
+- [ ] Ensure your venue is [accessible](/accessibility/)  
+- [ ] Create an Etherpad for your workshop at http://pad.software-carpentry.org/YYYY-MM-DD-site (where "YYYY-MM-DD-site" is the ID of your workshop)
 - [ ] [Set up your workshop website](https://github.com/datacarpentry/workshop-template)  
 - [ ] Set up your workshop registration page  
```

Comment:
maybe switch creating the workshop website and the registration page, since the registration info needs to go in the website